### DISCUSSION TITLE
Interacting/liking a status on a remote instance (by URL?)

### DISCUSSION THREAD

**Discussion by dato** [bookwyrm-social/bookwyrm#2990] [Q&A]
Given a status update like, say: https://lectura.social/user/SrMoshuelo/generatednote/35805

Is it possible to like it from my instance?

(For example, in Mastodon, I can just paste that URL as a search term, and it's recognized as an ActivityPub object that can be interacted with.)

If it simplifies things, let's say I've recently followed the user, but the backlog if their updates isn't on my instance.

And if it's not possible at the moment, any hints about what work (or related issues) would be needed to make it happen? I might have some time in the near future for a code contribution...

**Comment by Zepfanman** [comment]
I would really like to be able to do the same. Usually, there aren't many other local readers or reviews on books I'm reading. It would be nice to be able to interact across instances. It looks like the only thing I can do is follow another ActivityPub account from BookWyrm, but there's no other way to interact with that account, as far as I can tell.
